src/components/Person/Person.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  partnerName,
+  isMarried,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age ? (<p className="Person__age">{`I am ${age}`}</p>) : ('')}
+    <p className="Person__partner">
+      {isMarried ? (
+        `My
+        ${sex === 'm' ? ' wife\'s ' : ' husband\'s '}
+        name is ${partnerName}`
+      ) : (
+        'I am not married'
+      )}
```

Comment:
краще виносити так дії в окрему змінну за межами return, щоб можна було читати легше
```suggestion
      {isMarried ? (
        `My ${var} name is ${partnerName}`
      ) : (
        'I am not married'
      )}
```